Storefront/CartViewController.swift

```diff
@@ -193,92 +233,110 @@ extension CartViewController: TotalsControllerDelegate {
     
     func totalsController(_ totalsController: TotalsViewController, didRequestPaymentWith type: PaymentType) {
         let cartItems = CartController.shared.items
-        Client.shared.createCheckout(with: cartItems) { checkout in
-            guard let checkout = checkout else {
-                print("Failed to create checkout.")
-                return
-            }
-            
-            let completeCreateCheckout: (CheckoutViewModel) -> Void = { checkout in
-                switch type {
-                case .webCheckout:
-                    self.openSafariFor(checkout)
-                    
-                case .applePay:
-                    Client.shared.fetchShopName { shopName in
-                        guard let shopName = shopName else {
-                            print("Failed to fetch shop name.")
-                            return
-                        }
-                        
-                        self.authorizePaymentFor(shopName, in: checkout)
-                    }
+        if (type == .shopPay) {
```

Comment:
We should be able to remove the parentheses here:

```suggestion
        if type == .shopPay {
```